src/nft/components/view/NFTNetwork.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactNode } from 'react';
+import { baseSvg } from '../../../internal/svg/baseSvg';
+import { cn, text } from '../../../styles/theme';
+import { useOnchainKit } from '../../../useOnchainKit';
+
+type NFTNetworkProps = {
+  className?: string;
+  label?: string;
+};
+
+const networkMap = {
+  Base: baseSvg,
+} as Record<string, ReactNode>;
+
+export function NFTNetwork({ className, label = 'Network' }: NFTNetworkProps) {
+  const { chain } = useOnchainKit();
```

Comment:
I think we should deprecate this top-level `chain` prop, so I don't recommend using it here. 

Is there any other way to tell which chain the NFT is on?